pyproject.toml

```diff
@@ -16,28 +16,19 @@ omegaconf = "^2.1.1"
 # Why do we modify the requirements?
 # This framework is intended for running common deep learning models for point cloud analysis.
 # The original requirements are quite diverse and are not suited for our purposes (i.e. running KPConv in OSS).
-# Hence, we strip out some of the dependencies that are not needed for our use case.
```

Comment:
Why is this removed? Also the other lines?

Reply:
There are some deprecated statements there, and I think the kept part is already telling the story:
```
# Why do we modify the requirements?
# This framework is intended for running common deep learning models for point cloud analysis.
# The original requirements are quite diverse and are not suited for our purposes (i.e. running KPConv in OSS).
```

Reply:
> # We set the versions for torch-{cluster, scatter, sparse} fixed here because you need torch installed before poetry can build these. 
> # See https://github.com/rusty1s/pytorch_scatter/issues/252. We have to use specific wheels instead.

Are these statements deprecated?

Reply:
Yes kind of, the project is moved to PyG source instead (see [this comment](https://github.com/promaton/torch-points3d/pull/8#discussion_r1784235848) from Sean) + the mentioned issue is closed.

Reply:
Alright cool :+1: